linera-execution/src/runtime.rs

```diff
@@ -969,6 +969,7 @@ impl<UserInstance> BaseRuntime for SyncRuntimeInternal<UserInstance> {
                     })?
                     .recv_response()?
             };
+        self.resource_controller.track_http_request()?;
```

Comment:
If we have multiple HTTP requests within a single operation/proposal - won't this charge multiples? i.e. 1. First `perform_http_request` call will charge for 1 request
2. Second call will charge for 2
3. Third call will charge for 3

In total, instead of charging 3 multiples we would charge 6 🤔

Reply:
No, there's a test for that in the unit tests (it includes three HTTP requests).